src/main/java/xades4j/verification/SignaturePolicyVerifier.java

```diff
@@ -94,6 +95,7 @@ public QualifyingProperty verify(
                 sigDocStream.close();
             } catch (IOException ex)
             {
+                // TODO: remove throwing exceptions in finally block

```

Comment:
This is not a finally block. Revert comment?

Reply:
the whole try catch is inside a finally block. ;-)